tools/phenomenal/w4m/update-tools

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Fantastic, can you do that since your have this branch locally, and push that change? then we are set to go!